src/GraphQL/GraphQLBuilderExtensions.cs

```diff
@@ -915,7 +915,9 @@ public static IGraphQLBuilder ConfigureSchema(this IGraphQLBuilder builder, Acti
         /// </remarks>
         public static IGraphQLBuilder ConfigureExecutionOptions(this IGraphQLBuilder builder, Action<ExecutionOptions> action)
```

Comment:
Obsolete or not? What is our desing now to configure execuution options? Modify it into new `ConfigureExecution` method?

Reply:
The same question for ` public static IGraphQLBuilder ConfigureExecutionOptions(this IGraphQLBuilder builder, Func<ExecutionOptions, Task> action)`.

Reply:
I would leave `ConfigureExecutionOptions` for convenience if nothing else.  In v6 it will build an instance of `IConfigureExecution` rather than `IConfigureExecutionOptions`.  However, as I realized during our own tests, for compatibility in the case of a custom document executer's constructor arguments, it must continue to build `IConfigureExecutionOptions` during v5.

Reply:
ok